infra/feast-operator/test/e2e_rhoai/feast_wb_test.go

```diff
@@ -0,0 +1,138 @@
+/*
+Copyright 2025 Feast Community.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2erhoai
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	utils "github.com/feast-dev/feast/infra/feast-operator/test/utils"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Feast Jupyter Notebook Testing", Ordered, func() {
+	const (
+		namespace       = "test-ns-feast-wb"
+		configMapName   = "feast-wb-cm"
+		rolebindingName = "rb-feast-test"
+		notebookFile    = "test/e2e_rhoai/resources/feast-test.ipynb"
+		pvcFile         = "test/e2e_rhoai/resources/pvc.yaml"
+		notebookPVC     = "jupyterhub-nb-kube-3aadmin-pvc"
+		testDir         = "/test/e2e_rhoai"
+		notebookName    = "feast-test.ipynb"
+		feastMilvusTest = "TestFeastMilvusNotebook"
+	)
+
+	BeforeAll(func() {
+		By(fmt.Sprintf("Creating test namespace: %s", namespace))
+		Expect(utils.CreateNamespace(namespace, testDir)).To(Succeed())
+		fmt.Printf("Namespace %s created successfully\n", namespace)
+	})
+
+	AfterAll(func() {
+		By(fmt.Sprintf("Deleting test namespace: %s", namespace))
+		Expect(utils.DeleteNamespace(namespace, testDir)).To(Succeed())
+		fmt.Printf("Namespace %s deleted successfully\n", namespace)
+	})
+
+	runNotebookTest := func() {
+		env := func(key string) string {
+			val, _ := os.LookupEnv(key)
+			return val
+		}
+
+		username := utils.GetOCUser(testDir)
+
+		// create config map
+		By(fmt.Sprintf("Creating Config map: %s", configMapName))
+		cmd := exec.Command("kubectl", "create", "configmap", configMapName, "-n", namespace, "--from-file="+notebookFile, "--from-file=test/e2e_rhoai/resources/feature_repo")
```

Comment:
I think we should set the namespace context initially, so that you dont need tp set it in all commands.
```
kubectl config set-context --current --namespace <namespace_name>
```

Reply:
After this remove `-n` from all below commands unless you are chaing the namespace or reffering to another.